src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -1,10 +1,30 @@
-from typing import List, Optional, Tuple, Dict, Any
-
+from typing import List, Optional, Tuple
 import numpy as np
 from scipy.sparse import coo_matrix
-
+import sys
 from ophys_etl.transforms.data_loaders import motion_border
 
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+
+class StandardROI(TypedDict):
```

Comment:
It's not a typing class, but I actually have this structure as a marshmallow schema somewhere in here...

Reply:
Oh, yeah you also have this structure as a marshmallow schema. We should definitely figure out a place to consolidate. Some module in ophys_etl, `schemas` or `typing` or something.

Reply:
Will wait